src/node_modules.cc

```diff
@@ -344,6 +360,28 @@ void BindingData::GetNearestParentPackageJSONType(
   args.GetReturnValue().Set(Array::New(realm->isolate(), values, 3));
 }
 
+void BindingData::GetPackageJSONScripts(
```

Comment:
I absolutely don't see the point of doing this from C++ "this late".

 - If all it does is read a JSON file it can live in JS and be roughly as fast.
 - If we do it in C++, why bootstrap everything, spin up an isolate and do all that? I figured the point of doing it in C++ is to do a simple (sync) read (ideally without needing a uv threadpool?), parse the json and execute the child process.

I'm not blocking of that but the approach here seems like the worst of both worlds?

Reply:
I'll follow up and rewrite it in
C++

Reply:
To be clear I am not blocking this (and will lgtm, when the remaining doc fixes are fixed) - just trying to understand the motivation

Reply:
motivation: speed. more.